spec/app_spec.rb

```diff
@@ -101,16 +103,17 @@
       expect(last_response.body).to include(email)
     end
 
-    it 'has a logout link back to root, clicking it logs the user out' do
+    it 'has a logout link back to the SP-initiated logout URL' do
       get '/auth/result', code: code
 
       doc = Nokogiri::HTML(last_response.body)
 
-      logout_link = doc.at('a[href="/"]')
-      expect(logout_link.text.strip).to eq('Log out')
+      logout_link = doc.at_xpath("//a[text()='Log out']")
+      expect(logout_link).to be
```

Comment:
No, I just wanted to make sure it wasn't `nil` so that the next line wouldn't break